dev/integration_tests/external_ui/android/app/src/main/java/com/yourcompany/externalui/MainActivity.java

```diff
@@ -0,0 +1,141 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
```

Comment:
ideally we'd use /io/flutter/... rather than /com/yourcompany/...

Reply:
Done.